mne/time_frequency/tfr.py

```diff
@@ -1065,7 +1065,7 @@ def save(self, fname, overwrite=False):
 def _prepare_write_tfr(tfr, condition):
     """Aux function"""
     return (condition, dict(times=tfr.times, freqs=tfr.freqs,
-                            data=tfr.data, info=tfr.info, nave=tfr.nave,
+                            data=tfr.data, info=tfr.info._store, nave=tfr.nave,
```

Comment:
is this really needed?

to me it suggests that the new object breaks the API/interface of the previous Info object.